posix.mak

```diff
@@ -308,7 +308,7 @@ IMAGES=favicon.ico $(ORGS_USING_D) $(addprefix images/, \
 		d3 dconf_logo_2018 debian_logo dlogo fedora_logo freebsd_logo gentoo_logo homebrew_logo \
 		index-banner nix_logo \
 		opensuse_logo ubuntu_logo windows_logo pattern github-ribbon \
-		dlogo_opengraph \
+		dlogo_opengraph snapcraft_logo \
```

Comment:
Have you checked/confirmed the obligations/permissions for using the snapcraft logo?

Reply:
They don't provide any branding guidelines (I couldn't find any when I looked).
Anyhow, it's generally allowed to use trademarked logos if the customer isn't confused or a negative image is created towards the company.
Now that snapcraft itself is GPL licensed and we do "free advertisement" for them, I doubt that this is a problem. After all, they want that people use their logo like e.g. here: https://hexchat.github.io/downloads.html